CONTRIBUTING.md

```diff
@@ -14,6 +14,40 @@ Before sending your pull requests, make sure you do the following:
 -   Run the [unit tests](#running-unit-tests).
 
 ## How to become a contributor and submit your own code
+![Screen Shot 2022-08-30 at 7 27 04 PM](https://user-images.githubusercontent.com/42785357/187579207-9924eb32-da31-47bb-99f9-d8bf1aa238ad.png)
+
+### Typical Pull Request Workflow - 
+**1. New PR**
+   - As a contributor, you submit a New PR on GitHub.
```

Comment:
We are not talking about pre-submit an issue. Is a connected issue mandatory or at least strongly suggested?

It could help to lower the stats of "unwanted PRs" or PRs where we don't want to allocate review resources. 
Considering that the policy of allocating community reviews on the PRs is still unclear I think it is important to clarify this point.